go/src/koding/kites/config/konfig.go

```diff
@@ -170,6 +170,7 @@ type Konfig struct {
 	PublicBucketRegion string `json:"publicBucketRegion,omitempty"`
 
 	LockTimeout time.Duration `json:"lockTimeout,omitempty"`
+	NoMetrics   bool          `json:"noMetrics,string,omitempty"`
```

Comment:
> why just not:

Typically in Go you try to make zero values useful - if you have a configuration struct, and default behaviour is that something is enabled, you need to resort to `NoSomething` or `DisableSomething` in order to make zero value meaningful.